e2e/scenario/clusterconfiguration.go

```diff
@@ -14,10 +18,30 @@ func NetworkPluginKubenetSelector(cluster *armcontainerservice.ManagedCluster) b
 	return false
 }
 
+func NetworkPluginAzureSelector(cluster *armcontainerservice.ManagedCluster) bool {
+	if cluster != nil && cluster.Properties != nil && cluster.Properties.NetworkProfile != nil {
+		return *cluster.Properties.NetworkProfile.NetworkPlugin == armcontainerservice.NetworkPluginAzure
+	}
+	return false
+}
+
 // Mutators
 
 func NetworkPluginKubenetMutator(cluster *armcontainerservice.ManagedCluster) {
 	if cluster != nil && cluster.Properties != nil && cluster.Properties.NetworkProfile != nil {
 		cluster.Properties.NetworkProfile.NetworkPlugin = to.Ptr(armcontainerservice.NetworkPluginKubenet)
 	}
 }
+
+func NetworkPluginAzureMutator(cluster *armcontainerservice.ManagedCluster) {
```

Comment:
Looks like this will always set default. value no matter what. Could we highlight that fact in the function name?

Reply:
oh, err, do you mean on NetworkPluginKubenetMutator? since kubenet is the default network plugin